test-integration/scores.coffee

```diff
@@ -0,0 +1,127 @@
+{describe, CourseSelect, screenshot} = require './helpers'
+selenium = require 'selenium-webdriver'
+{expect} = require 'chai'
+
+TEACHER_USERNAME = 'teacher01'
+
+MAIN_ELEMENTS = [
+  '.scores-report',
+  '.course-scores-wrap',
+  '.nav-tabs',
+  '.export-button',
+  '.course-scores-title',
+  '.course-scores-container'
+]
+
+
+class StudentScores
+
+  constructor: (@test, @login, @courseTitle) ->
```

Comment:
Why include login in the constructor? This seems like it would be difficult to do something like:

- login
- test that there are at least 2 courses
- pick a course
- go to student scores